src/BlazorWebView/src/Maui/Android/BlazorWebViewHandler.Android.cs

```diff
@@ -26,6 +26,8 @@ protected override AWebView CreateNativeView()
 #pragma warning restore 618
 			};
 
+			AWebView.SetWebContentsDebuggingEnabled(enabled: true);
```

Comment:
Is an `#if DEBUG` too simplistic of a check here?  Are there scenarios we want to enable debugging perhaps via some configuration or xplat api to explicitly enable it at runtime?

Reply:
End-users will need to be able to debug, so we can't condition it on our own compilation, because users will never get the DEBUG version.

But yeah we'll need to eventually figure out a way to determine it based on the _user's_ desire. It might just be some setting/event on the control.